MOM6-interface/CMakeLists.txt

```diff
@@ -23,11 +23,13 @@ set(MOM6SOLO OFF CACHE BOOL "Build MOM6 solo")
 
 # Too many files to list, so include them via this file
 include("mom6_files.cmake")
-
+#SP LIB
+# sp::sp_d
 ### Use common object library for building target library
 add_library(mom6_obj OBJECT ${mom6_src_files})
 set_target_properties(mom6_obj PROPERTIES Fortran_MODULE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/mod)
-target_include_directories(mom6_obj PRIVATE $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/mod>)
+target_include_directories(mom6_obj PRIVATE $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/mod>
```

Comment:
Are these CMakeList changes necessary if we are not committing the MOM6 stochastic options at this time?

Reply:
They are not necessary, but I had it working for when I thought the MOM6 stochastic physics PR was going in with this.